test/examples/change_streams.js

```diff
@@ -35,11 +36,20 @@ describe('examples(change-stream):', function() {
 
       // Start Changestream Example 1
       const collection = db.collection('inventory');
+
+      // process as a stream
```

Comment:
`changeStream` is not a `ReadableStream`, merely an `EventEmitter`. This example would more accurately be called `// process with event API`. Process as a stream would be using `changeStream.pipe()`. And we should explicitly NOT advertise `changeStream.stream()` b/c it does bad things.

Reply:
updated the examples to remove these comments